### DISCUSSION TITLE
These namings are reversed

### DISCUSSION THREAD

**Discussion by forest008** [community/community#75321] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I believe that VS code and visual studio is inverted in https://github.com/features/copilot?
<img width="896" alt="image" src="https://github.com/community/community/assets/83619817/dcec1303-260a-4a65-a06e-9bc8a90d449b">

**Comment by sky9262** [comment, marked as answer]
It has been fixed!!!!

**Comment by sky9262** [comment]
![image](https://github.com/community/community/assets/68050118/985afd91-bbd3-4e63-b04b-76376e2aa124)

**Comment by ettaboyle** [comment]
Thanks for calling this out @forest008 and closing the loop @sky9262 ♥️ 🚀
